tests/testthat/test-gather_fails.R

```diff
@@ -0,0 +1,469 @@
+library(shiny)
+library(shinyvalidate)
```

Comment:
Don't add library here - if the packages are in the depends/suggests/imports then it's not needed

Reply:
Unless it's in Depends, should it not be attached for the example to work?

Reply:
So if its in suggests or imports then you can use `shinyvalidate::` inside the tests 

![image](https://user-images.githubusercontent.com/15201933/205301298-07d396bf-f9f2-4888-84e9-433af356b584.png)

we don't need to guard against using suggests functions in the tests, in the same way we don't need to for the examples  for R cmd check to pass

Reply:
Pardon me, I phrased my question poorly, I meant "for the example to work in its current form".
Yes, the namespace prefix precludes the need for attaching but that is what I explicitly wanted to avoid. It seems using `library` in examples is common practice, no?

Reply:
in examples yes, but in test files no?